src/brouwers/shop/payments/sisow/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from ...models import Payment, ShopConfiguration
+from .api import calculate_sha1
+from .constants import TransactionStatuses
+from .service import get_ideal_banks
+
+
+def coerce_bank(value):
+    bank_mapping = {bank.id: bank for bank in get_ideal_banks()}
+    return bank_mapping[value]
+
+
+class CallbackForm(forms.Form):
```

Comment:
Maybe add a short comment what this callback form is for, since it's not completely clear from the code ( at least to me).

Reply:
I'll add a docstring 👍 

FYI, after the payment Sisow either calls us async or redirects us back, and it processes the querystring parameters of that callback to process the payment state on our side again.